docs/automated-testing/getting-started.md

```diff
@@ -0,0 +1,125 @@
+The Controller also provides APIs for spawning bots and managing recordings (via a separate, Automation Recorder component attached to the same object).
+
+To add the Automation Controller to your scene, click **GameObject > Regression Games > Automation Controller** in the Unity menu.
+
+![The "Automation Controller" option in the Regression Games submenu of the GameObject menu](img/getting-started/add-automation-controller-menu.png)
+
+**By default**, the Automation Controller and all it's children will mark themselves as [DontDestroyOnLoad](https://docs.unity3d.com/ScriptReference/Object.DontDestroyOnLoad.html). This ensures that bots continue to run when you load a new scene in your game. You can disable this behavior by unchecking the "Dont Destroy On Load" checkbox on the Automation Controller component.
+
+![The Automation Controller component with the "Dont Destroy On Load" checkbox checked](img/getting-started/ac-dont-destroy-on-load.png)
+
+## Adding Entity Discoverers to your scene
+
+In order to automate entities in your game, the Automation Controller needs to be able to find them.
+The Automation Controller uses Entity Discoverers to find entities in the scene that can be automated.
+The package provides a few built-in Entity Discoverers, but you can also create your own.
+To add one of the built-in discoverers, open the **GameObject > Regression Games > Discovery** menu and select the discoverer you want to add:
+
+![The "UI Element Discoverer" option in the Regression Games submenu of the GameObject menu](img/getting-started/add-uielement-discoverer.png)
+
+As long as you have an Automation Controller in your scene, the Entity Discoverer will be **automatically** be added as a child of the Automation Controller, no matter what object is selected in the Hierarchy window.
+
+![The Unity Hierarchy view showing the "UI Element Discoverer" object as a child of the "Automation Controller"](img/getting-started/ui-element-discoverer-in-scene.png)
+
+For this example, we recommend spawning the "UI Element Discoverer", which will find all the UI elements in your scene.
+Elements will provide various states that can be recorded, such as the opacity and interactability of a Canvas Group.
+Some elements provide actions that the bot can take, such as clicking on a Button.
+
+## Adding the Overlay UI
+
+For simple scenarios, we provide a UI Overlay that can be used to spawn bots in the scene.
+This allows you to quickly try out automations without having to write code to manage the bots.
+To add the Overlay UI, click **GameObject > Regression Games > UI Overlay** in the Unity menu.
+
+![The Overlay UI running in the Boss Room sample](img/getting-started/overlay-menu.png)
+
+The UI Overlay provides a small button in the bottom-right corner of the game.
+When clicked, this button will open a dialog that shows you a list of available bots that can be spawned.
```

Comment:
Yeah, because they're separate concerns. The automation _engine_ doesn't depend on the Overlay UI. There are APIs on Automation Controller to start/stop bots and handle recordings. The Overlay UI is purely a UI for interacting with those APIs without having to write custom code. I could see a production game having the Automation Controller but **not** having the UI Overlay.